parm/config/config.base.emc.dyn

```diff
@@ -121,8 +121,9 @@ export RUN=${RUN:-${CDUMP:-"gfs"}}  # RUN is defined in the job-card (ecf); CDUM
 # TODO: determine where is RUN actually used in the workflow other than here
 # TODO: is it possible to replace all instances of ${CDUMP} to ${RUN} to be
 #       consistent w/ EE2?
-export COMIN_OBS=${DMPDIR}/${CDUMP}.${PDY}/$cyc/atmos
-export COMIN_GES_OBS=${DMPDIR}/${CDUMP}.${PDY}/$cyc/atmos
+export COMIN_OBS=${COMIN_OBS:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/obs}
+export COMIN_GES_OBS=${COMIN_GES_OBS:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/obs}
```

Comment:
Is there a reason this needs to have a variable?  This is a configuration file and nothing is set before calling `config.base`
```suggestion
export COMIN_OBS="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/obs"
export COMIN_GES_OBS="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/obs"
```